packages/block-library/src/post-tags/edit.js

```diff
@@ -34,9 +40,31 @@ function PostTagsDisplay() {
 	);
 }
 
-export default function PostTagsEdit() {
-	if ( ! useEntityId( 'postType', 'post' ) ) {
-		return __( 'Post Tags' );
+export default function PostTagsEdit( { context } ) {
+	if ( ! context.postType || ! context.postId ) {
+		return (
+			<Warning>
+				{ __( 'Post tags block: No post found for this block.' ) }
+			</Warning>
+		);
+
+		/**
+		 * Do not render the block when viewing a page (as opposed to a post)
+		 *
+		 * @todo By default, only posts can be grouped by tags. Therefore, without any configuration,
+		 * the post tags block will have no tags for pages. Plugins, however, can modify this behavior.
+		 * In the future, instead of only evaluating posts, we should check whether the
+		 * post_tag taxonomy is registered for the current post type.
+		 */
```

Comment:
Hmm, I wish `prettier` didn't indent this comment so much :thinking:

Reply:
Yeah it felt odd to me as well. I wonder if we can change some rules...lemme check.

Reply:
After some poking around, it seems like the easiest way to solve this is with `prettier-ignore` comments. https://prettier.io/docs/en/ignore.html They don't seem to work in the block-library though 🤔

Reply:
It looks like `prettier-ignore` doesn't work on comment formatting 😅  I think we may be stuck with the extra indentations.

Reply:
I gave this some more thought. For inspiration, I looked into the anatomy of a pre-FSE theme (Twenty Twenty):

![image](https://user-images.githubusercontent.com/96308/88530973-6f0f3100-d002-11ea-936d-9806ee1d660b.png)

A lot of these come with a number of class name conventions (Post Content's `.entry-content` being one of the more popular ones; or those `.meta-...` ones). However, the particular nesting of `<div />`s and attached class names are fairly theme-specific; after all, they depend on the visual structure of the theme.

It's tempting to carry over some of those class names so that block-based themes can be styled. But that might actually be a fallacy; after all, we want themes to be editable in the Site Editor, which means that all styling should be customizable through the block's controls (and block inspector sidebar, and Global Styles), rather than manual CSS editing.

Maybe we should give this block a light block wrapper though, since that seems to be the sort of interface we tend to base some of this customizability on.